.travis.yml

```diff
@@ -29,9 +37,18 @@ before_install:
   - go get -u golang.org/x/lint/golint
   - go env && pwd
   - mkdir -p $HOME/gopath/bin
-script:
+
+install:
```

Comment:
install is for setting up prerequisites. it is ok to have this as the first instruction under script.

Reply:
updated